reconcile/ocm_upgrade_scheduler.py

```diff
@@ -33,14 +35,18 @@ def fetch_current_state(clusters, ocm_map):
     return current_state
 
 
-def fetch_desired_state(clusters):
+def fetch_desired_state(
+    clusters: list[dict[str, Any]], ocm_map: OCMMap
+) -> list[dict[str, Any]]:
```

Comment:
i'm just trying to figure out how this works hand in hand with `ocm-clusters`.

`ocm-clusters` creating/managing clusters and setting the `channel` based on the contents of `/openshift/clusters-1.yml` and `ocm-upgrade-policy` consuming that information from the OCM API instead from a cluster file or from the `upgradePolicyClusters[*].spec.channel` of an `/openshift/openshift-cluster-manager-1.yml`

correct?

Reply:
correct.